Sail/src/Sail/entities/systems/Gameplay/LevelSystem/LevelSystem.cpp

```diff
@@ -1772,14 +1772,20 @@ void LevelSystem::addClutterModel(const std::vector<Model*>& clutterModels, Mode
 
 #ifdef DEVELOPMENT
 		auto* particleEmitterComp = e2->addComponent<ParticleEmitterComponent>();
-		particleEmitterComp->size = 1.0f;
+
+		float sprinklerXspread = room.sizex * tileSize * 1.3f;
+		float sprinklerZspread = room.sizey * tileSize * 1.3f;
+
+		particleEmitterComp->size = 0.1f;
 		particleEmitterComp->offset = { 0, tileHeight * 6, 0 };
-		particleEmitterComp->constantVelocity = { 0.0f, -0.7f, 0.0f };
-		particleEmitterComp->acceleration = { 0.0f, -0.4f, 0.0f };
-		particleEmitterComp->spread = { 5.22f, 1.0f, 5.22f };
-		particleEmitterComp->spawnRate = 1.f / 100.f;
```

Comment:
Static divisions should instead be static multiplications, as multiplying is a lot cheaper than division.
   1.f / 100.f   
should be:
   1.f * 0.001f

Reply:
This comment is made on removed code.